sentences/nl/fan_HassTurnOff.yaml

```
@@ -5,13 +5,17 @@ intents:
       - sentences:
           - "<doe> [alle] <ventilator> [<naar>] uit in <area>"
           - "<doe> [alle] <ventilator> in <area> [<naar>] uit"
-          - "<doe> (alle | in) <area> <ventilator> [<naar>] uit"
+          - "<doe> (alle|in) <area> <ventilator> [<naar>] uit"
         response: fans_area
         slots:
           domain: "fan"
           name: "all"
       - sentences:
-          - "<doe> (overal | alle) <ventilator> uit"
+          - "<doe> <name>[' ']<ventilator> [<naar>] uit"
+        slots:
```

Gebruik requires_context zodat je alleen names gelinkt aan fan domain matched.

```suggestion
        requires_context:
          domain: "fan"
        slots:
```